ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetDetailedObjects.java

```diff
@@ -38,8 +38,8 @@
 import java.util.Map;
 import javax.annotation.Nullable;
 
-public class GetFilteredObjects extends CliCommand<GetDetailedObjectsResult> {
-    private final static org.slf4j.Logger LOG = LoggerFactory.getLogger(GetFilteredObjects.class);
+public class GetDetailedObjects extends CliCommand<GetDetailedObjectsResult> {
+    private final static org.slf4j.Logger LOG = LoggerFactory.getLogger(GetDetailedObjects.class);
```

Comment:
Command name change should be updated in resources: com.spectralogic.ds3cli.command.CliCommand and com.spectralogic.ds3cli.help.properties